ansible/roles/systemd/templates/pulp_content_app.j2

```diff
@@ -0,0 +1,15 @@
+[Unit]
```

Comment:
@bmbouter Is the content app meant to be run as a systemd unit?

Reply:
That is a fine way to run it. End users could select that or other choices, whatever aiohttp supports https://aiohttp.readthedocs.io/en/stable/deployment.html